tracee/tracee.bpf.c

```diff
@@ -233,8 +235,8 @@ BPF_HASH(chosen_events_map, u32, u32);              // Events chosen by the user
 BPF_HASH(pids_map, u32, u32);                       // Save container pid namespaces
 BPF_HASH(args_map, u64, args_t);                    // Persist args info between function entry and return
 BPF_HASH(ret_map, u64, u64);                        // Persist return value to be used in tail calls
-BPF_HASH(uid_equal_filter, u32, u32);               // Used to filter events by UID, for specific UIDs either by == or !=
-BPF_HASH(uid_compare_filter, u32, u32);             // Used to filter events by UID, for ranges of UIDs by < or >
+BPF_HASH(uid_equal_filter, u32, int64_t);             // Used to filter events by UID, for specific UIDs either by == or !=
```

Comment:
Do we need this to be 64 bits long or can we keep it 32?

Reply:
They need to be signed 64 bits because they can be -1 or u32_max +1

Reply:
Hm, actually I put validation in the CLI portion of the code to make the user specify valid UIDs. Think it's worth it to support -1 and u32_max+1? Otherwise there _shouldn't_ ever be none u32 values here.